ledger/acctonline.go

```diff
@@ -633,7 +635,8 @@ func (ao *onlineAccounts) lookupOnlineAccountData(rnd basics.Round, addr basics.
 				// Check if this is the most recent round, in which case, we can
 				// use a cache of the most recent account state.
 				if offset == uint64(len(ao.deltas)) {
-					return macct.data.OnlineAccountData(rewardsProto, rewardsLevel), nil
+					out = macct.data.OnlineAccountData(rewardsProto, rewardsLevel)
```

Comment:
It feels more complex to me to have set named output variables from from inside a closure, inside a helper function, versus just being able to return them directly to the caller as before ...